docs/deploy-controller.md

```diff
@@ -16,25 +16,25 @@ The CSI-Addons Controller can be deployed by different ways:
 
 > Note: Some of the above configuration options can also be configured using [`"csi-addons-config"` configmap](./csi-addons-config.md).
 
+## Installation for latest deployments
+
+The latest CSI-Addons Controller can be installed using the YAML files in `deploy/controller` from `main` branch.
+
 ## Installation for versioned deployments
 
-The CSI-Addons Controller can also be installed  using the YAML files in `deploy/controller`.
-The versioned deployment is possible with the YAML files that get generated for the
-[latest release](https://github.com/csi-addons/kubernetes-csi-addons/releases/latest).
-You can download the YAML files from there, or use them directly with kubectl.
-This is the recommended and easiest way to deploy the controller.
+The versioned deployment of the CSI-Addons Controller can be installed using the specific version's YAML files as shown below:
```

Comment:
Can you please add a line for the user to update the RELEASE to the required release, Below show is just an example for latest release, replace `v0.8.0` with `latest` so that we don't need to keep updating it later

Reply:
> Can you please add a line for the user to update the RELEASE to the required release, 

This should be good enough.